### PR TITLE
#166272090 Integrate travis-ci 

### PR DESCRIPTION
**Pull request by Sojisoyoye**
## What does this PR do?

This PR Integrates travis ci to run build

#### Description of Task to be completed?

- Setup the project on travis-ci website
- Add .travis.yml file 
- Add necessary configuration settings on travis.yml file
- Add travis build/status badge

#### How should this be manually tested?

No manual test. The test is automated after each push on travis website

#### Any background context you want to provide?

This is setting up continuous integration with travis-ci

#### What are the relevant pivotal tracker stories?

[Finishes #166272090]

### CODE REVIEW

**Review by EmmanuelSage — APPROVED** [review]
Good job Soji,
it looks okay to me.

**Review by frankly034 — APPROVED** [review]
Nice job Soji. Looks good.

**Review by Eazybee — APPROVED** [review]
Looks good to me. Good job

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+langauage: node
```
Comment (benfluleck) [inline .travis.yml:1]: I expect you to have Travis integration for slack please set it up

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - postgresql
+
+install:
+  - npm install
+
+script:
+  - npm test
+
+before_script:
+  - psql -c 'create database kingkongah;' -U postgres
```
Comment (benfluleck) [inline .travis.yml]: test database should be sqlite not postgres

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+install:
+  - npm install
+
+script:
+  - npm test
+
+notifications:
+  slack:
+    secure: C39KywRCO97fSN5UqAcpamWH724a3Kv97176npKpQ38b8gFR0CGxUrtsi/QaK4rwVgyMJIo/vVuirgem6szlHf52jctvJIExukXLQGJDWUB+Lhr+gOmTECutcaKhOfLod790eY+L06FJCWYSfMiFLbB+ET/0UNJgtafWqi5aqZB1zqv1e3s2L+cGgF8pZ4XvboY5bC4y/Sfn7QYj/0L6uht0zlhItqd5GgMwNr2ygON6HbPHDnV0vdb/6Th+fFzU3GFe79Vwa2l3kowZVxIEtxqBoIgS7szoHUL+cq6xWQIrSG0lKg5i/XpQ/5wxaQXT9o8yB+owJRbWZCef7UVvJkssJeBeSb1tyij5ETPp3ryFGEuVZg2LHKiwa2m6OiRbf5ieTKq0gXD8ZoHOEa9SOtyvPpCc4ogurJ6RLY5AbYJxHfV8x4pANxzav75dxVa3iDXht4FJePoYDel/nxIAEgPQ99mMBnoGK8aVKZFGT9QBuOfmRG3j4BIDbbEgrtpAHWLWas9QuqYN4U1Pzq9JcgeOwUzoAGo3rzTT9wlEYjxODHQmkFI+juV30Q45mEekvNs58ws6crPrf32N3xW9o8XU9Gl4hHZZavsRcbZ+rZ5D4JsR2pvvAHqjbVMHDJdanpRugEgUB1sleutHo5jZ0DeYKq5S3qobqD9SD7fEVC8=
```
Comment (benfluleck) [inline .travis.yml]: Find a way to put this token in .env. I told you to talk to Ndifreke

Comment (Sojisoyoye) [reply .travis.yml]: I already message him. He's yet to reply to me

Comment (Sojisoyoye) [reply .travis.yml]: At your request to put the token in .env - I got to realise that Travis does not support that feature yet according to (travis-ci/travis-ci#7201) and (travis-ci/travis-ci#6387) issues, rather the token can be secured by running Travis encrypt command on the generated token, thereby producing an encrypted string of the token which can now be passed into notifications as I did above.

**Comment by benfluleck** [comment]
Sort out your conflicts @Sojisoyoye

**Comment by benfluleck** [comment]
I told you to sort out your conflicts. Remove the label after you do this

**Pull request merged**